### PR TITLE
Place battle results at top of log

### PR DESCRIPTION
**Pull request by Kookiecraft**
temporary fix for virtual scrolling being bad

### CODE REVIEW

File: src/plugins/combat/battle.js [hunk 1]
```diff
@@ -203,6 +207,14 @@ export class Battle {
   endBattle() {
     this._emitMessage('Battle complete.', this._partyStats());
     this.endBattleBonuses();
+    
+    //temporary fix for battle log virtual scrolling
+    this.results.reverse().forEach(result => {
+      this.messageData.splice(1, 0, this.results);
+    });
+    this.messageData.splice(1, 0, "Summary");
+    console.log(this.messageData);
```
Comment (seiyria) [inline src/plugins/combat/battle.js]: please remove console.log

**Comment by Kookiecraft** [comment]
Should be already removed in the most recent commit

**Comment by seiyria** [comment]
ah, my bad. only saw one at the time. when it passes we should be good!

**Comment by Kookiecraft** [comment]
Dang it! CI so strict

**Comment by seiyria** [comment]
thanks. I'll merge this, hopefully it works!

**Pull request merged**